### PR TITLE
Updated build instructions for os x

### PR DESCRIPTION
**Pull request by ishankhare07**
new instructions using:
1. Qt Creator
2. cmake

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -32,46 +32,41 @@ make -j4
 And that should give you `rbkit-app/RBKitClient` executable, which you can
 go ahead and run.
 
-## Qt build issue on OSX Mavericks
```
Comment (emilsoman) [inline README.md]: Why did you remove the note about the build issue on Mavericks ?

File: README.md [hunk 2]
```diff
@@ -32,46 +32,41 @@ make -j4
...
-look for `-spec` argument. You need to add a flag `QMAKE_MACOSX_DEPLOYMENT_TARGET = 10.9`
-in the `qmake.conf` file of that spec.
+1. Using Qt Creator
+   Double click the `rbkit-client/RbkitClient.pro` file to open it in the qt creator. In the left menu-bar click on `Debug` button and select `rbkit-app` under the Run field.  
+   Hit the Run button, after compilation the GUI should pop-up.
 
-For example:
+2. Using cmake
+   Running a shadow build, i.e building from a separate directory is very
+simple. Just create a build directory, say `~/rbkit-build`. Lets say
```
Comment (emilsoman) [inline README.md]: Not the best example - nobody wants to put `rbkit-build` in home !

File: README.md (hunk 1 above)
Comment (ishankhare07) [inline README.md]: because i think its not more there, its solved because we can either load it directly in qt creator itself or run the cmake to build. I didn't encounter any issues by using the above commands

Comment (ishankhare07) [inline README.md]: I didn't do all these fixes, it worked without them as well.

File: README.md [hunk 3]
```diff
@@ -32,46 +32,41 @@ make -j4
...
+   ```
 
-*Note* - Above special syntax is usually not required on Linux systems.
+   run cmake
 
-Running a shadow build, i.e building from a separate directory is very
-simple. Just create a build directory, say `~/rbkit-build`. Lets say
-source code is cloned at `~/rbkit-client`
+   ```bash
+   $ cmake -DZMQ_BUILD_TESTS=OFF ../rbkit-client
```
Comment (emilsoman) [inline README.md]: @iffyuva Sorry I'm not familiar with cmake, but is there a way to set `-DZMQ_BUILD_TESTS=OFF` by default ? And possibly also a default build path , which can be changed with `--prefix` make style ? That way , we can just go to the project directory and simply run `cmake` and we're done.

File: README.md (hunk 2 above)
Comment (iffyuva) [inline README.md]: i generally create a directory where i cloned rbkit-client.

File: README.md (hunk 3 above)
Comment (iffyuva) [inline README.md]: we can always edit CMakeLists.txt of zeromq, and turn this off. I haven't explored how to do it from top level CMakeLists.txt file. @ishankhare07 can you explore this?

cmake has to be run from build directory, just like how we run `configure` script from build directory. i don't thing there is an easy way to provide a default build right now.

File: README.md (hunk 2 above)
Comment (ishankhare07) [inline README.md]: this example is as it is, just replaced its position a bit. If we need a directory like `Desktop`, I will update it with it

Comment (ishankhare07) [inline README.md]: @iffyuva  done with #165

File: README.md (hunk 3 above)
Comment (ishankhare07) [inline README.md]: @iffyuva done with #165
